RELEASING.md

```diff
@@ -20,23 +20,24 @@ that a release candidate will be created, published and tested before the final
 release is announced.
 
 1. Run release-scripts/do_release.py to copy master directory to the new version directory
-2. Add the new version to `_data/versions.yaml` (add the date of the release in the description)
+1. Add the new version to `_data/versions.yaml` (add the date of the release in the description)
    - Add new versions that are a release candidate to the end of the file.
      If a release is added to the top of the file it will become the 'Latest Release'.
-3. Add a section in `_config.yaml` so that `page.version` will be set correctly in the new subdirectory:
-````
+1. Add a section in `_config.yaml` so that `page.version` will be set correctly in the new subdirectory:
+```
```

Comment:
While you're here, can you indent this code block three spaces so it doesn't reset the numbered list?